network/topics/params/peer_score.go

```diff
@@ -45,7 +45,7 @@ func PeerScoreThresholds() *pubsub.PeerScoreThresholds {
 }
 
 // PeerScoreParams returns peer score params according to the given options
-func PeerScoreParams(oneEpoch, msgIDCacheTTL time.Duration, ipWhilelist ...*net.IPNet) *pubsub.PeerScoreParams {
+func PeerScoreParams(oneEpoch, msgIDCacheTTL time.Duration, disableCoLocation bool, ipWhilelist ...*net.IPNet) *pubsub.PeerScoreParams {
```

Comment:
Let's keep the same naming style as in `IPColocationFactorWeight`

```suggestion
func PeerScoreParams(oneEpoch, msgIDCacheTTL time.Duration, disableColocation bool, ipWhilelist ...*net.IPNet) *pubsub.PeerScoreParams {
```